docs/deprecated_scss-linting/.stylelintrcComments

```diff
@@ -0,0 +1,227 @@
+
+    "function-calc-no-unspaced-operator": true,                 // possible error                           ( stylelint-config-recommended )    -> stylelintscssrc
+
+    "function-comma-space-after": "always",                     // stylistic issue                          ( stylelint-config-standard )       -> stylelintscssrc
+                                                                                                              always-single-line
+
+    "function-url-no-scheme-relative": true,                    // limit feature                                                                -> stylelintscssrc
+
+    "function-url-quotes": "always",                            // stylistic issue                                                              -> stylelintscssrc
+
+    "indentation": 4,                                           // stylistic issue                          ( stylelint-config-standard )       -> stylelintscssrc
+                                                                                                              2
+
+    "max-nesting-depth": 3,                                     // limit feature                                                                -> stylelintscssrc
+
+    "no-duplicate-selectors": true,                             // possible error                                                               -> stylelintscssrc
+
+    "no-extra-semicolons": true                                 // possible error                           ( stylelint-config-recommended )    -> stylelintscssrc
+
+    "no-missing-end-of-source-newline": true,                   // stylistic issue                          ( stylelint-config-standard )       -> stylelintscssrc
+
+    "number-leading-zero": "always",                            // stylistic issue                          ( stylelint-config-standard )       -> stylelintscssrc
+
+    "number-no-trailing-zeros": true,                           // stylistic issue                          ( stylelint-config-standard )       -> stylelintscssrc
+
+    "property-no-vendor-prefix": true,                          // limit feature                                                                -> stylelintscssrc
+
+    "selector-class-pattern": "^[a-z\\-_0-9]+$",                // limit feature                                                                -> stylelintscssrc
+
+    "selector-no-qualifying-type": true,                        // limit feature                                                                -> stylelintscssrc
+
+    "shorthand-property-no-redundant-values": true,             // possible error                           ( stylelint-config-recommended )    -> stylelintscssrc
+
+    "string-quotes": "single",                                  // stylistic issue                                                              -> stylelintscssrc
+  }
+}
```

Comment:
This file is of ne need, please remove it.

Reply:
File removed.